src/microbe_stage/Patch.cs

```diff
@@ -23,6 +23,12 @@ public class Patch
     [JsonProperty]
     public readonly int[] Depth = new int[2] { -1, -1 };
 
+    /// <summary>
+    ///   The volume of the patch, in meters cube.
```

Comment:
Do we really want to restrict ourselves to exact units here?

Reply:
Tough question that has been haunting my nights for a while (do I exaggerate here? Yes, definitely.).

Depth itself is in meters, and so the volume, defined as deltaDepth^3 is in meters cube. 
I feel like having real units helps getting a clear view of what values represent physically, but I agree it is a restriction.

I don't have much of an opinion on this.